### PR TITLE
chore(protocol): remove tokenomics and format solidity file (#13822)

### LINKED ISSUE #13833: LibBridgeRetry backward gasLimit check comments vs code
### Describe the bug

In the `retryMessage()` function in [LibBridgeRetry.sol](https://github.com/taikoxyz/taiko-mono/blob/main/packages/protocol/contracts/bridge/libs/LibBridgeRetry.sol), the check for situation in which the owner must be the caller is backwards from what's specified in the comments.

```
// If the gasLimit is not set to 0 or isLastAttempt is true, the
// address calling this function must be message.owner.
if (message.gasLimit == 0 || isLastAttempt) {
    if (msg.sender != message.owner) revert B_DENIED();
}
```

### Steps to reproduce

N/A

### Additional context

I _think_ the code is correct and the comment is backwards (it would make more sense to me that `gasLimit = 0` would denote owner only), but not positive, so wanted to submit just in case.

### PR DESCRIPTION
**Pull request by dantaik**
This PR is to prepare for the introduction of a new tokenomics. @davidtaikocha  going forward, please submit all tokenomics tweak directly onto alpha-3 branch.

When review this PR, please turn on "hide whitespace".

Screenshot 2023-05-27 at 16 27 21

### CODE REVIEW

File: packages/protocol/contracts/L1/TaikoErrors.sol [hunk 1]
```diff
@@ -7,7 +7,8 @@
 pragma solidity ^0.8.18;
 
 abstract contract TaikoErrors {
-    // The following custom errors must match the definitions in other V1 libraries.
+    // The following custom errors must match the definitions in other V1
+    // libraries.
     error L1_ALREADY_PROVEN();
     error L1_BLOCK_ID();
     error L1_EVIDENCE_MISMATCH(bytes32 expected, bytes32 actual);
```
Comment (davidtaikocha) [inline packages/protocol/contracts/L1/TaikoErrors.sol:14]: in this file, the following `L1_INSUFFICIENT_TOKEN` error should also be removed? or we just want to keep this error?

File: packages/protocol/contracts/L1/libs/LibProving.sol [hunk 2]
```diff
@@ -43,18 +43,28 @@ library LibProving {
...
-    ) internal {
+    )
+        internal
+    {
         if (
-            evidence.parentHash == 0 || evidence.blockHash == 0
-                || evidence.blockHash == evidence.parentHash || evidence.signalRoot == 0
-                || evidence.gasUsed == 0
+            evidence.parentHash == 0
+            //
```
Comment (davidtaikocha) [inline packages/protocol/contracts/L1/libs/LibProving.sol:51]: is this `//` added automatically by formatter? hmm not sure if this looks better?

File: packages/protocol/contracts/L1/libs/LibVerifying.sol [hunk 3]
```diff
@@ -74,12 +75,15 @@ library LibVerifying {
...
+    )
+        internal
+    {
         uint256 i = state.lastVerifiedBlockId;
         TaikoData.Block storage blk = state.blocks[i % config.ringBufferSize];
 
         uint256 fcId = blk.verifiedForkChoiceId;
-        assert(fcId > 0);
+        // assert(fcId > 0);
+        require(fcId > 0, "ABC");
```
Comment (davidtaikocha) [inline packages/protocol/contracts/L1/libs/LibVerifying.sol:86]: is this "ABC" error message what we want?

File: packages/protocol/contracts/L1/libs/LibVerifying.sol [hunk 4]
```diff
@@ -92,7 +96,8 @@ library LibVerifying {
         address systemProver = resolver.resolve("system_prover", true);
         while (i < state.numBlocks && processed < maxBlocks) {
             blk = state.blocks[i % config.ringBufferSize];
-            assert(blk.blockId == i);
+            // assert(blk.blockId == i);
+            require(blk.blockId == i, "EFG");
```
Comment (davidtaikocha) [inline packages/protocol/contracts/L1/libs/LibVerifying.sol:100]: ditto

File: packages/protocol/foundry.toml [hunk 5]
```diff
@@ -28,4 +28,7 @@ fs_permissions = [
 fuzz = { runs = 256 }
 
 [fmt]
-line_length = 100
+line_length = 80
+multiline_func_header = 'all'
+number_underscore = 'thousands'
+wrap_comments = true
```
Comment (davidtaikocha) [inline packages/protocol/foundry.toml:34]:
```suggestion
wrap_comments = true

```

File: packages/protocol/contracts/L1/TaikoErrors.sol (hunk 1 above)
Comment (dantaik) [reply packages/protocol/contracts/L1/TaikoErrors.sol:14]: Done

File: packages/protocol/contracts/L1/libs/LibVerifying.sol (hunk 3 above)
Comment (dantaik) [reply packages/protocol/contracts/L1/libs/LibVerifying.sol:86]: Removed

**Review by davidtaikocha — DISMISSED** [review]

File: packages/protocol/contracts/L1/libs/LibProving.sol (hunk 2 above)
Comment (dantaik) [reply packages/protocol/contracts/L1/libs/LibProving.sol:51]: I just want to make sure each condition is put in its own line, this is the only way I can make it happen.

**Review by davidtaikocha — DISMISSED** [review]

**Review by davidtaikocha — APPROVED** [review]

File: packages/protocol/contracts/L1/TaikoData.sol [hunk 6]
```diff
@@ -135,19 +131,19 @@ library TaikoData {
         // Slot 7: never or rarely changed
         uint64 genesisHeight;
         uint64 genesisTimestamp;
-        uint16 adjustmentQuotient;
+        uint16 __reserved70;
```
Comment (dantaik) [inline packages/protocol/contracts/L1/TaikoData.sol:134]: Reviewers, please make sure you check the data layout shall be compatible with alpha-3

**Review by dionysuzx — APPROVED** [review]

**Pull request merged**